app/models/account_migration.rb

```diff
@@ -14,6 +14,8 @@ class AccountMigration < ApplicationRecord
   attr_accessor :old_private_key
   attr_writer :old_person_diaspora_id
 
+  attr_accessor :archive_contacts
```

Comment:
Style/AccessorGrouping: Group together all `attr_accessor` attributes.

Reply:
This is a new check, which wasn't there in the past (at most places in the code they aren't grouped), and I suggest disabling it, because I don't find it useful. Hacing the attributes separated and being able to order/group them freely makes the code better structured in my opinion.

Reply:
I agree with it.